src/credentials/FabricTable.cpp

```diff
@@ -1715,7 +1729,7 @@ CHIP_ERROR FabricTable::AddNewPendingFabricCommon(const ByteSpan & noc, const By
 
     // Notify that NOC was added (at least transiently)
     *outNewFabricIndex = fabricIndexToUse;
-    NotifyFabricUpdated(fabricIndexToUse);
+    NotifyFabricAdded(fabricIndexToUse);
```

Comment:
Changed the function `OnFabricUpdated()` to `OnFabricAddedOrUpdated()` in the Delegate, so that the existing API consumers will know the API changes and change their implement.